pkg/cmd/build/build_docker.go

```diff
@@ -171,12 +171,16 @@ func buildWithDockerDaemon(ctx context.Context, buildOptions *types.BuildOptions
 
 	dockerBuildContext, err := getBuildContext(buildOptions.File)
 	if err != nil {
-		errors.Wrap(err, "setting build context failed")
+		if err := errors.Wrap(err, "setting build context failed"); err != nil {
+			oktetoLog.Infof("failed to set error: %s", err)
+		}
 	}
 
 	dockerBuildOptions, err := getDockerOptions(buildOptions)
 	if err != nil {
-		errors.Wrap(err, "setting build options failed")
+		if err := errors.Wrap(err, "setting build options failed"); err != nil {
+			oktetoLog.Infof("failed to set error: %s", err)
+		}
```

Comment:
@jLopezbarb these 2 errors were not being handled before, they were just being wrapped without returning them. 

Should we return them now?